### PR TITLE
path commands fix and improve

### PR DESCRIPTION
**Pull request by Ebmtranceboy**

### CODE REVIEW

**Comment by JordanMartinez** [comment]
@Ebmtranceboy I apologize. Apparently, I wasn't watching this repo (likely because it was a fork?) so I was never notified of your PR.

**Comment by Ebmtranceboy** [comment]
no problem!

File: src/Halogen/Svg/Indexed.purs [hunk 1]
```diff
@@ -78,6 +78,7 @@ type SVGg = GlobalAttributes
 type SVGpath = MarkerAttributes (GlobalAttributes
   ( d :: String
   , transform :: String
+  , strokeWidth :: Number
```
Comment (JordanMartinez) [inline src/Halogen/Svg/Indexed.purs:81]: Could you remove this one line? Once #12 gets merged, it'll be included here anyway and that means there won't be a merge conflict if I merge this one first.

**Review by JordanMartinez — COMMENTED** [review]
LGTM. Just one minor change.

File: src/Halogen/Svg/Indexed.purs (hunk 1 above)
Comment (Ebmtranceboy) [reply src/Halogen/Svg/Indexed.purs]: done!

**Comment by JordanMartinez** [comment]
Thanks!

**Comment by JordanMartinez** [comment]
CI failure is due to using a more recent version of spago. It should be fixed in #12, so I'll go ahead and merge this.

**Pull request merged**